css/style.css

```diff
@@ -840,3 +889,10 @@ ul.chars{
     top: 50%;
     transform: translate(-50%, -50%);
 }
+
+code {
+    border: 0.1em solid darkgrey;
+    border-radius: 0.25em;
+    padding: 0.1em;
+    background: black;
+}
```

Comment:
you made the file no longer end in a newline, but the rest of them do iirc - you many want this one to as well